pa11ycrawler/settings.py

```diff
@@ -101,5 +19,15 @@
 # See http://scrapy.readthedocs.org/en/latest/topics/item-pipeline.html
 ITEM_PIPELINES = {
     'pa11ycrawler.pipelines.DuplicatesPipeline': 200,
+    'pa11ycrawler.pipelines.DropDRFPipeline': 250,
     'pa11ycrawler.pipelines.Pa11yPipeline': 300,
 }
+
+# Other items you are likely to want to override ---------------
+CONCURRENT_REQUESTS = 16
```

Comment:
Concurrency is built into Scrapy. Relevant settings are here: http://doc.scrapy.org/en/1.1/topics/settings.html#concurrent-items

This item was already in the `settings.py` file, I just moved things around so Git is a bit confused and is marking it as a new addition.